internal/service/privatelinkendpointservicedatafederationonlinearchive/data_source_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -13,6 +13,7 @@ var (
 )
 
 func TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchiveDS_basic(t *testing.T) {
+	acc.SkipTestForCI(t)
```

Comment:
optionally you can add a comment why it's skipped to help in the future in case we want to run it in CI